example/app.py

```diff
@@ -69,22 +81,64 @@ async def test_postgres_changes(socket: AsyncRealtimeClient):
     # Add your access token here
     # await socket.set_auth("ACCESS_TOKEN")
 
-    channel = socket.channel("test-postgres-changes")
+    channel: AsyncRealtimeChannel = socket.channel("test-postgres-changes")
+
+    def on_subscribe(status: RealtimeSubscribeStates, err: Optional[Exception]) -> None:
+        subscribed = True
+        print(f"ON_SUBSCRIBE: {status} (Error: {err})")
 
     await channel.on_postgres_changes(
         "*", table="todos", callback=postgres_changes_callback
     ).on_postgres_changes(
         "INSERT",
         table="todos",
-        filter="id=eq.10",
+        filter="description=eq.test",
         callback=postgres_changes_insert_callback,
     ).on_postgres_changes(
-        "DELETE", table="todos", callback=postgres_changes_delete_callback
+        "DELETE",
+        table="todos",
+        callback=postgres_changes_delete_callback,
     ).on_postgres_changes(
-        "UPDATE", table="todos", callback=postgres_changes_update_callback
-    ).subscribe()
+        "UPDATE",
+        table="todos",
+        callback=postgres_changes_update_callback,
+    ).subscribe(
+        on_subscribe
+    )
+
+    asyncio.create_task(socket.listen())
+
+    # Wait a short time to ensure we are properly listening
+    await asyncio.sleep(1)
 
-    await socket.listen()
+    headers = {"Prefer": "return=representation"}
+
+    # does not match filter and will therefore only be received by the * listen, but not the INSERT listen
+    requests.post(
```

Comment:
Am I correct in thinking the `request` library is used here purely for testing purpose? if yes then I don't think this should be in the example as users will blindly copy this without understanding what is going on. Rather instructions should be left in a README on how to run the example and call the endpoints from a HTTP GUI client or CURL.

Reply:
Yes, @silentworks, the example is only for testing purposes. It is there to have a quick app to run and test the realtime implementation, actually. But thinking more about that, maybe it is better to remove this example and use an integration test instead.

What do you think?

Reply:
Yeah if its for testing purpose only then it should be in the tests directory as an integration test instead. This is how we've structured the `postgrest-py` repo, it has normal unit tests and integration tests inside of the tests directory.

Reply:
@silentworks can you review again? I decided to remove the example then, and keep the integration tests that already do the same.